### ISSUE TITLE
Envato Theme Updater

### ISSUE THREAD

**Issue by Gnodesign** [richtabor/MerlinWP#29]
Hey Richard,

Hope you are doing great.

First of all I want to thank you for doing an amazing job as an Envato Theme Reviewer and for this great project.

I played a little with the importer and it's really amazing. Can't wait for the official release to add it to my theme. It gives the user a whole different experience. When do you think of releasing it?

I also wanted to ask you, do you think of adding a theme auto updater as [Envato WordPress Theme Setup Wizard](https://github.com/dtbaker/envato-wp-theme-setup-wizard)? It would really be a great feature for this importer.

Best regards,
Gino

**Comment by Gnodesign** [comment]
Is this importer still in development?

**Comment by richtabor** [comment]
Yes, though abiet slowly.

**Comment by primozcigler** [comment]
Hey @Gnodesign our team has recently helped @richtabor on the Merlin WP and we're getting closer to the v1. The importer has been improved dramatically, read more about it here: https://primoz.blog/pushing-merlin-wp-towards-stable-release/

Please test out the Merlin WP if you have time.

**Comment by lavanta** [comment]
I am working on this feature, but I can't test it right now. We do not have any items on ThemeForest. I can not send a request to this end-point **'/v3/market/author/sale?code='**. Well I can verify my own purchases through **'/v3/market/buyer/purchase?code='**.

We have chosen the Laravel Framework for the API server, so you can easily extend it. 

I need someone who can help me. I am also stuck with some other points, we can discuss this later. But first, who is able to test this feature with real sale codes?

**Comment by AdelDima** [comment]
@lavanta Do you have any pull on this, I can test it I have items on ThemeForest.

**Comment by lavanta** [comment]
@AdelDima No, I don't have any pull on this. I stopped working on it, but maybe I can start again in the future. I will let you know.
